core/src/UserSettingsDialog.html

```diff
@@ -379,114 +447,238 @@
     border: none;
   }
 
-  .iframeUserSettingsCtn {
-    overflow: visible;
-    -webkit-overflow-scrolling: touch;
+  .lui-fd-side__nav-wrapper {
+    height: 75vh;
+    max-height: 35rem;
+    overflow: auto;
   }
 
-  .lui-usersettings-content-header__backBtn {
+  .lui-usersettings-content-header__back-btn {
     display: none;
-    float: left;
-    width: 0;
-  }
-
-  .lui-usersettings-content-header {
-    position: -webkit-sticky;
-    position: sticky;
-    top: 0;
-    width: 100%;
-    height: 50px;
   }
 
-  .lui-usersettings-content-header__title {
-    width: 100%;
-    font-size: 16px;
-    font-weight: bold;
-    text-align: center;
-    line-height: 50px;
+  .lui-usersettings-dialog-sub-header {
+    position: relative;
+    left: var(--left-fd-side-nav-width);
+    border-bottom: none;
   }
 
-
   .usersettingseditor {
     padding: 20px;
   }
 
   @media (max-width: 600px) {
-    .dialog-size {
-      height: 80vh;
-    }
-
     :global(.usersettings-leftNavVisible) .lui-usersettings-left-nav {
       width: 0;
     }
+    /*micro frontend and iframe wrappers inside the right-side dialog body*/
+    .mf-wrapper,
+    .iframe-wrapper {
+      overflow: auto;
```

Comment:
When I remove the `overflow:auto` I don't get a double scrollbar on iframe mf.
Currently I have it? Does it make sense?